devel/Github-Desktop/Portfile

```diff
@@ -3,17 +3,17 @@
 PortSystem          1.0
 
 name                Github-Desktop
-version             2.9.6
+version             2.9.6-9196a1ae
 revision            0
 categories          devel
 platforms           darwin
 supported_archs     x86_64 arm64
 license             MIT 
-maintainers         openmaintainer usersxx
+maintainers         openmaintainer usersxx {@usersxx microsoft.com:usersword123}
```

Comment:
change this line to `maintainers        {@usersxx microsoft.com:usersword123} openmaintainer`